src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -145,6 +151,22 @@ def test_conditional_jump_triggers_rep_cap(multi_instrument_session):
     assert requested_trigger_type == multi_instrument_session.conditional_jump_triggers['conditionalJumpTrigger3'].conditional_jump_trigger_type
 
 
+def test_rio_events_rep_cap(single_instrument_session):
+    assert '' == single_instrument_session.rio_events['RIOEvent3'].exported_rio_event_output_terminal
+
+    requested_terminal_name = '/Dev1/PXI_Trig0'
+    single_instrument_session.rio_events['RIOEvent3'].exported_rio_event_output_terminal = requested_terminal_name
```

Comment:
Passing 3 should work too, as a number. Please test those cases.
Writing out the whole string is not the recommended way to program necessarilly.

Reply:
Good catch, totally forgot about that nice feature :).

Updated the test.